configure.ml

```diff
@@ -349,6 +349,8 @@ let args_options = Arg.align [
     " URL of the coq website";
   "-force-caml-version", Arg.Set Prefs.force_caml_version,
     " Force OCaml version";
+  "-camldir", Arg.String (fun _ -> ()),
+    "<dir> Specifies path to ocaml for running configure script";
```

Comment:
What about putting quotes around as done in `configure`? We are not talking here about the OCaml software but about the `ocaml` executable.